cmake/projects/breakpad/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    breakpad
+    VERSION
+    1.0-p0
+    URL
+    "https://github.com/t0p4/breakpad/archive/v1.0-p0.tar.gz"
```

Comment:
Please send pull request with changes to the fork: https://github.com/hunter-packages/breakpad

It make sense to deal with it before submitting main pull request.

Reply:
I didn't find information about it. what do you advise

Reply:
> I didn't find information about it. what do you advise

You mean there was no breakpad fork before? You can open "New package" issue and mention there that we need fork.

Reply:
I mean about package version